definitions/v7/lat-team-api.yml

```diff
@@ -113,24 +107,12 @@ search:
       selector: name:contains(VOSE)
       optional: true
       filters:
-        - name: re_replace
-          args: ["^ *\\[[^\\]]*\\] *", ""] # Remove prefix tags
-        - name: re_replace
-          args: ["(?i)\\bT(\\d+)", "S$1"]
-        - name: re_replace
-          args: ["UHDRip", "BDRip"] # Fix for Radarr
         - name: append
           args: " ENGLiSH"
     title_notvose:
       selector: name:not(:contains(VOSE))
       optional: true
       filters:
-        - name: re_replace
-          args: ["^ *\\[[^\\]]*\\] *", ""] # Remove prefix tags
```

Comment:
this is  for parsing the results returned

this is a non issue anymore and all prefixed tags for all existing entries have been removed?

Reply:
Lat-Team does not used prefixed tags.